### PR TITLE
[ar] Linear Algebra and Calculus 

### PR DESCRIPTION
**Pull request by zaidalyafeai**
A work in progress.

### CODE REVIEW

**Comment by zaidalyafeai** [comment]
@shervinea, I am done with the translation. It would be great if I can see how the page will render on the website to see if there are problems regarding mixing latex with the Arabic text.

**Comment by shervinea** [comment]
Great work @zaidalyafeai, thank you for your time! Don't worry about the mathematical rendering, we are going to take care of it.

Now, let's start the review process!

**Comment by zaidalyafeai** [comment]
@AmjadKhatabi, you can review this if you have some time. You might face some difficulties because of the latex formulae. You can just focus on the vocabulary.

File: ar/refresher-linear-algebra.md [hunk 1]
```diff
@@ -1,339 +1,413 @@
...
+<div dir="rtl">
+   المعكوس (Inverse)- معكوس أي مصفوفة $A$ قابلة للعكس (Invertible) يرمز له ب $A^{-1}$ ويعتبر المعكوس المصفوفة الوحيدة التي لديها الخاصية التالية :
+</div>
 <br>
 
 **25. Remark: not all square matrices are invertible. Also, for matrices A,B, we have (AB)−1=B−1A−1**
 
-&#10230;
+<div dir="rtl">
+ملاحظة: ليس جميع المصفوفات يمكن أيجاد معكوس لها. كذلك لأي مصفوفتين $A$ و $B$ نستنتج $(AB)^{-1} = B^{-1} A^{-1}$.
```
Comment (AmjadKhatabi) [inline ar/refresher-linear-algebra.md]: a very minor typo :) 
إيجاد

**Comment by zaidalyafeai** [comment]
Updated the initial review comments !

File: ar/refresher-linear-algebra.md [hunk 2]
```diff
@@ -1,339 +1,413 @@
 **1. Linear Algebra and Calculus refresher**
```
Comment (AmjadKhatabi) [inline ar/refresher-linear-algebra.md]: Finished the final checkups. The file is ready to go!

File: ar/refresher-linear-algebra.md [hunk 3]
```diff
@@ -1,339 +1,413 @@
...
 
-&#10230;
-
+<div dir="rtl">
+  ملخص عن الجبر الخطي
```
Comment (mznmel) [inline ar/refresher-linear-algebra.md]: Shouldn't this be: "ملخص الجبر الخطي و التفاضل و التكامل"?

File: ar/refresher-linear-algebra.md [hunk 4]
```diff
@@ -1,339 +1,413 @@
...
+ مصفوفة (Matrix) - نرمز ل ${A \in \mathbb{R}^{m\times n$ مصفوفة تحتوي على $m$ صفوف و $n$ أعمدة، حيث $A_{i,j}$  يرمز للمدخل في الصف$ i$ و العمود $j$  
+</div>
 
 <br>
 
 **6. Remark: the vector x defined above can be viewed as a n×1 matrix and is more particularly called a column-vector.**
-
-&#10230;
+<div dir="rtl">
+ملاحظة : المتجه $x$ المعرف مسبقا يمكن اعتباره مصفوفة من الشكل $n \times 1$ والذي يتم تسميته ب مصفوفة من عمود واحد.
```
Comment (mznmel) [inline ar/refresher-linear-algebra.md]: أقترح تعديل: "والذي يتم تسميته ب مصفوفة" إلى "و الذي يسمى مصفوفة من عمود واحد"

File: ar/refresher-linear-algebra.md [hunk 5]
```diff
@@ -1,339 +1,413 @@
...
+  الضرب
+</div>
 
 <br>
 
 **14. Vector-vector ― There are two types of vector-vector products:**
 
-&#10230;
+<div dir="rtl">
+  متجه و متجه - هناك نوعين من الضرب ل متجه - متجه :
```
Comment (mznmel) [inline ar/refresher-linear-algebra.md]: أقترح أن يترجم السطر بهذا الشكل:
"ضرب المتجهات: توجد طريقتين لضرب متجه بمتجه:"
بعيدة قليلاً عن الترجمة الحرفية لكن أظنها أسهل للفهم للقارئ العربي

File: ar/refresher-linear-algebra.md [hunk 6]
```diff
@@ -1,339 +1,413 @@
...
+</div>
 
 <br>
 
 **17. Matrix-vector ― The product of matrix A∈Rm×n and vector x∈Rn is a vector of size Rn, such that:**
 
-&#10230;
-
+<div dir="rtl">
+  مصفوفة - متجه : ضرب المصفوفة $A \in \mathbb{R}^{n\times m}$ والمتجه $x \in \mathbb{R}^n$ ينتجه متجه من الشكل $x \in \mathbb{R}^n$ حيث :
```
Comment (mznmel) [inline ar/refresher-linear-algebra.md]: نفس الملاحظة السابقة، أظن أسهل على القارئ العربي أن يترجم السطر بهذا الشكل:
"ضرب مصفوفة و متجه: ضرب مصفوفة .. و متجه .. ينتج عنه متجه .."

**Comment by zaidalyafeai** [comment]
@mznmel, thanks I made the changes. Regarding the arrow, the Persian don't contain any. Maybe @shervinea  can shed some light on that.

**Comment by zaidalyafeai** [comment]
@shervinea this has been revised twice. Shouldn't it be merged ?

**Comment by shervinea** [comment]
@zaidalyafeai Thanks for letting me know! Will move forward with the merge right now!

**Pull request merged**